extract_unique_sensors_01.py

```diff
@@ -0,0 +1,28 @@
+from urllib.request import urlopen
+
+sensors_data = urlopen('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+lines = sensors_data.readlines()
+
+
+def sensors(lines):
+    #global lines
+    candidate_lines = []
+    for line in lines:
+        line = line.decode()
+        if line.startswith('kat.sensors'):
+            for candidate_line in line.split():
+                candidate_lines.append(candidate_line)
+    return candidate_lines
+
+# Loop over candidate_lines and extract sensor names
+def sensor_names():
```

Comment:
declarative function name i.e summarize what your function is doing with one line.